smartcontracts/assets.go

```diff
@@ -40,6 +40,14 @@ var (
 		EthNetwork: types.ETHSepolia,
 		Version:    erc20token.ERC20TokenBase,
 	}
+	// Sandbox
+	TBTC_S VegaAsset = VegaAsset{
+		Name:       "tBTC",
```

Comment:
Can we change the var name to `VT_S` (like `Vega Token Sepolia`), and change the `tBTC` to `Vega Token Sepolia`